dmref_analyzer/FileBrowser.py

```diff
@@ -1,55 +1,62 @@
-import glob, os
+from _typeshed import StrOrBytesPath
+import glob
+import os
 import fnmatch
 import re
 import numpy as np
 import util
 
 import pandas as pd
 import sys
+from os.path import dirname, abspath
 
 
 class FileBrowser(object):
     def __init__(self, root_dir=None, param_rule_file=None):
         if root_dir is None:
-            self.root_dir = os.path.expanduser('~/Dropbox/DMREF/Database/')
+            self.root_dir = dirname(dirname(abspath(__file__)))
         else:
             self.root_dir = root_dir
         if param_rule_file is None:
             param_rule_file = os.path.expanduser(
-                '~/Dropbox/DMREF/parameter_rules.csv')
+                './parameter_rules.csv')
```

Comment:
1. expanduser is not needed if you don't use "~" in the path
2. parameter_rules.csv won't exist under "./" if you call the genDendro.py file from within the tests directory.

Reply:
You're right!
I changed this part to >>> param_rule_file = os.path.join(os.getcwd(), 'parameter_rules.csv')

how about now?

Reply:
you can try to print os.getcwd() when you are running an interactive python shell inside the tests directory - you will see that it returns the path of "tests". so it didn't really fix it - as a matter of fact, using os.getcwd() is equivalent to using "."

Reply:
But this is the code of FileBrowser.py, and it's located in the dmref_analyzer module.